docs/changes/config/17640.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add support for setting astropy's cache and config directories via the
+``ASTROPY_CACHE_DIR`` and ``ASTROPY_CONFIG_DIR`` environment variables.
```

Comment:
The change log entry should make it clear that the new variables have priority over the XDG variables.